installing/installing_ibm_power/installing-ibm-power.adoc

```diff
@@ -123,3 +123,4 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 * xref:../../post_installation_configuration/cluster-tasks.adoc#available_cluster_customizations[Customize your cluster].
 * If necessary, you can
 xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[opt out of remote health reporting].
+* The link:https://github.com/ocp-power-automation/rsct-operator[RSCT Operator] can be installed to enable RSCT on the cluster.
```

Comment:
can you add a link to the README at the end of the sentence? Something like "...to enable RSCT on the cluster. Installation instructions are provided link:https://github.com/ocp-power-automation/rsct-operator/blob/main/README.md[here]

Reply:
I'm fine with this as is, but just something to consider if you wind up pushing another revision

Reply:
Thanks Mike. Above change has been incorporated into the PR.